docs/snapshots.md

```diff
@@ -21,6 +21,35 @@ aggregate loading.
 
 Our example application [proophessor-do](https://github.com/prooph/proophessor-do) contains a snapshotting tutorial.
 
+## Using a different Serializer. 
+
+By default we use php build in serialize/unserialize methods to around aggregate persistence. If you would like to change it you can do that since v1.1. 
```

Comment:
By default prooph uses PHP's own serialise and unserialize methods. These may not suite your needs so as of v1.1 of the snapshot store you can now use a custom serialiser.